src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/__tests__/test_suites/validation.command.stats.ts

```diff
@@ -170,7 +172,7 @@ export const validationStatsCommandTestSuite = (setup: helpers.Setup) => {
               'from a_index | stats avg(doubleField), percentile(doubleField, 50) + 1 by ipField',
               []
             );
-            await expectErrors('from a_index | stats ?func(doubleField)', []);
+            await expectErrors('from a_index | stats avg(doubleField) by ??field', []);
```

Comment:
Why this change?

Reply:
Because it is tested above, but the by is not tested so I decided to add this case here too